app/models/rdv_solidarites/rdv.rb

```diff
@@ -22,6 +22,10 @@ def participations
       end
     end
 
+    def participation_for(user)
+      participations.find { _1.user.id == user.rdv_solidarites_user_id }
```

Comment:
```suggestion
      participations.find { _1.user_id == user.rdv_solidarites_user_id }
```
Pour ne pas trigger une requête supplémentaire

Reply:
J'avais pas vu que c'était sur le model `rdv_solidarites` my bad 😄 je ferme le commentaire